onnx/optimizer/passes/eliminate_unused_initializer.h

```diff
@@ -0,0 +1,66 @@
+// ATTENTION: The code in this file is highly EXPERIMENTAL.
+// Adventurous users should note that the APIs will probably change.
+
+#pragma once
+
+// Before:
+//   A = Const
```

Comment:
We are not able to remove unused Const :-)
To be more precise,  you can say `A, B, C are in the initializer list`

Reply:
@houseroad initializer could not be a const?

Reply:
kConstant means you have a `Constant` op, see https://github.com/onnx/onnx/blob/master/docs/Operators.md#constant

kParam means the initializers, I think.

Reply:
@houseroad Maybe we should clarify them in future.